docs/rules.adoc

```diff
@@ -0,0 +1,46 @@
+ifdef::env-github,env-browser[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+:outfilesuffix: .adoc
+endif::[]
+
+=== Rules
+:icons: font
+
+If you want to use arquillian functionality with some other runner. You can use Arquillian with Junit Rule using `ArquillianTestClass` and `ArquillianTest` Rules.
+
+In order to use full functionality of Arquillian you have to use both the rules i.e. `ArquillianTestClass` and
+`ArquillianTest`.
+
+==== How to use it?
+
+[source, java, indent=0]
+----
+public class HelloWorldTest
+{
+   @ClassRule
+   public static ArquillianTestClass arquillianTestClass = new ArquillianTestClass();
+
+   @Rule
+   public ArquillianTest arquillianTest = new ArquillianTest();
+
+   @Test
+   public void shouldNotBeNull()
+   {
+      Assert.assertNotNull(arquillianTest);
+   }
+}
+----
+
+IMPORTANT:
+However while using above Junit Rules you can't inject method scoped arquillian resources.
+In order to use any other JUnit Method Rule with `ArquillianTest` Rule, you have to use it with RuleChain
```

Comment:
> \`RuleChain\`

Reply:
Maybe also worth to mention that unfortunately it is not possible to have one rule acting as both test class and test method rule https://github.com/junit-team/junit4/issues/351#issuecomment-102084524